### ISSUE TITLE
Debug view is not the welcome view, so I see no continue on...

### ISSUE THREAD

**Issue by sbatten** [microsoft/vscode#167606]
Testing #167433

insiders.vscode.dev

<img width="274" alt="image" src="https://user-images.githubusercontent.com/6561887/204619866-c4e7ac17-7b3a-43b7-9d6d-90c96a0ec951.png">

**Comment by joyceerhl** [comment]
Given we now do support debugging in the web with https://github.com/microsoft/vscode/issues/167399, I think the debug welcome view is never going to show up...or could the fact that the debug view is always populated now be related to workspace trust changes for web?

**Comment by sbatten** [comment]
There are no recent changes to workspace trust in the web that I am aware of, but it could be that we just assumed debugging is available now. It seems wrong for this case because I have launch configs, but they will not work as they require a different debug adapter.

**Comment by joyceerhl** [comment]
@connor4312 do you happen to have context on why the debug view is populated with launch configs in web now and not before?

**Comment by joyceerhl** [comment]
Ah, that'll be the experimental Python WASM extension, that makes sense. Also confirmed that if I uninstall that extension the welcome view reappears. @sbatten do you have that extension installed, and if you uninstall it does the debug view show up for you?

**Comment by sbatten** [comment]
I don't but it's actually coming from the main python extension. I disable that and get the welcome view. So as far as continue working on... this is working.

**Comment by joyceerhl** [comment]
@paulacamargo25  does the Python extension contribute a debugger that works in web, and if it does not, can we ensure not to register it in web?

**Comment by paulacamargo25** [comment]
@joyceerhl No, our debugger doesn't work on the web, I check and we already show a message, that python configuration is not supported.

<img width="699" alt="Screenshot 2022-11-30 at 11 17 25 AM" src="https://user-images.githubusercontent.com/17892325/204898444-deb0ef41-ab90-4a5d-872e-c9cd8ea69b09.png">

I'll check if it's possible not to register it at all.
